src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -100,6 +107,17 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
                 ResourceTypeIdsSetBuilder.of().addCategories())
             .build();
 
+    final StateDraft stateDraft =
```

Comment:
Drafts that are created in this method will be the exactly the same for every test case of this testsuite. Wouldn't it be better if we create all the draft objects only once for this testsuite and use them whenever we want? Because we do not change anything in drafts inside this testsuite. wdyt?

Reply:
We could, but let's not share data across the tests. Additionally it's not part of this task :)